frontend/src/components/history-page/sort-button/sort-button.tsx

```diff
@@ -64,13 +57,17 @@ const toggleDirection = (direction: SortModeEnum) => {
  * @param onDirectionChange Callback that is fired when the sorting direction is changed
  * @param direction The sorting direction that is used
  */
-export const SortButton: React.FC<SortButtonProps> = ({ children, variant, onDirectionChange, direction }) => {
+export const SortButton: React.FC<SortButtonProps> = ({ children, onDirectionChange, direction }) => {
   const toggleSort = () => {
     onDirectionChange(toggleDirection(direction))
   }
 
   return (
-    <IconButton onClick={toggleSort} variant={variant} icon={getIcon(direction)} border={true}>
+    <IconButton
+      onClick={toggleSort}
+      variant={direction === SortModeEnum.no ? 'outline-light' : 'light'}
```

Comment:
![image](https://user-images.githubusercontent.com/52606896/221177834-97d01679-7b58-4a68-a207-877afd25a250.png)

I'm not sure about this design decision yet. Isn't there any other possibility to indicate which sort button is active and which not?

Reply:
I changed it